tools/Makefile

```diff
@@ -0,0 +1,304 @@
+# docker-compose container image variables
+DEV_DOCKER_TAG_BASE ?= ghcr.io/ansible
+GIT_BRANCH ?= $(shell git rev-parse --abbrev-ref HEAD)
+COMPOSE_TAG ?= $(GIT_BRANCH)
```

Comment:
Maybe I'm the only one that creates branches using slashes for the prefix separator (i.e. `feature/my_branch`) but any branch name containing characters that are no allowed in image tags will cause this to fail. I normalized the tag in my experimental [PR #13337](https://github.com/ansible/awx/pull/13337/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R13) like this:

     COMPOSE_TAG ?= $(shell echo $(GIT_BRANCH) | sed "s/[^A-Za-z0-9.-]/_/g")